src/control/StrictControl.ts

```diff
@@ -242,9 +242,15 @@ class StrictControl extends Control {
       targetPanel = adjacentAnchor!.panel;
       targetPos = adjacentAnchor!.position;
     } else {
-      // Restore to active panel
-      targetPos = camera.clampToReachablePosition(activePanel.position);
-      targetPanel = activePanel;
+      // Fallback to nearest panel from current camera
+      const anchorAtCamera = camera.findNearestAnchor(camera.position);
+      if (!anchorAtCamera) {
+        return Promise.reject(new FlickingError(ERROR.MESSAGE.POSITION_NOT_REACHABLE(position), ERROR.CODE.POSITION_NOT_REACHABLE));
+      }
```

Comment:
nit: `findNearestAnchor` always returns a value if at least one panel exists, so basically there're no cases that `anchorAtCamera` doesn't exist.

Reply:
camera.findNearestAnchor method returns nullable, but moveToPanel does not accept nullable params.
Is it okay to non-null assert the return?

Reply:
You're right, that's typed the same as it behaves.
I think it's okay just as it is, in both cases, they'll throw errors but it's easier to find such circumstances when we throw errors like that.